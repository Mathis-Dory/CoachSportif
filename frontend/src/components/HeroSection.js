import React from 'react';
import '../App.css';
import { Button } from './Button';
import {Link} from "react-router-dom";
import './HeroSection.css';
import {isLoggedIn} from './auth.js';
import {Sidebar} from './sidebar'
function HeroSection() {

    if(isLoggedIn()){
        return (

        <div className='hero-container'>
          <img src='/images/img-02.jpg' />

          <h1>WORK HARDER, GET STRONGER</h1>
          <p>What are you waiting for?</p>

        </div>)
    }else{
        return(
        <div className='hero-container'>
          <img src='/images/img-02.jpg' />
          <h1>WORK HARDER, GET STRONGER</h1>
          <p>What are you waiting for?</p>
          <div className='hero-btns'>
            <Button
              className='form-input-btn'
            ><Link to="/sign-up">
              S'INSCRIRE</Link>
            </Button>

          </div>
        </div>)
    }

}

export default HeroSection;
