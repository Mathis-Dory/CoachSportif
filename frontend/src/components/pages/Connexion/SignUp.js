import React from 'react';
import '../../../App.css';
import FormSignup from "../../FormSignUp"

export default function SignUp() {
    return (
        <>
            <FormSignup/>;
        </>

    );
}
