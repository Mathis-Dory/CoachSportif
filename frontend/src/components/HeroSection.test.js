import React from "react";
import ReactDOM from "react-dom";
import HeroSection from "./HeroSection";
import { MemoryRouter } from "react-router-dom";
import TestRenderer from "react-test-renderer";
import SignUp from "./pages/Connexion/SignUp";

test("render sans que Ã§a plante", () => {
  const div = document.createElement("div");
  ReactDOM.render(<MemoryRouter><HeroSection /></MemoryRouter>, div);
});

it("renders a div", () => {
  const testRenderer = TestRenderer.create(<MemoryRouter><HeroSection /></MemoryRouter>);
  const testInstance = testRenderer.toJSON();
  expect(testInstance).toMatchSnapshot();
});