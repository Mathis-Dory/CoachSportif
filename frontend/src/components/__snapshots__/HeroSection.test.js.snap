// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a div 1`] = `
<div
  className="hero-container"
>
  <img
    src="/images/img-02.jpg"
  />
  <h1>
    WORK HARDER, GET STRONGER
  </h1>
  <p>
    What are you waiting for?
  </p>
  <div
    className="hero-btns"
  >
    <a
      className="btn-mobile"
      href="/login"
      onClick={[Function]}
    >
      <button
        className="btn btn--primary btn--medium"
      >
        <a
          href="/sign-up"
          onClick={[Function]}
        >
          S'INSCRIRE
        </a>
      </button>
    </a>
  </div>
</div>
`;
